### ISSUE TITLE
Dependencies ?

### ISSUE THREAD

**Issue by neven10** [stsrki/Blazorise#24]
Are dependencies for this  0.7.0 (AspNetCore.Blazor.Build and AspNetCore.Blazor.Browser)?
I'm trying to run default client side Blazor template with  Blazorise.Bootstrap package but something is exploding in Blazor.MonoRuntime.targets.

Also there's no AddIconProvider().

**Comment by stsrki** [comment]
Just tried it myself. For some reason there is no Blazorise dependency on NuGet. You will need to manually add NuGet package for Blazorise. Also you will need to add usings in the _Startup.cs_ eg.

```
using Blazorise;
using Blazorise.Bootstrap;

...

public void ConfigureServices( IServiceCollection services )
{
    services
        .AddBootstrapProviders()
        .AddIconProvider( IconProvider.FontAwesome );
}
```

That should work.

I will see what I can do regarding the NuGet packages...

**Comment by stsrki** [comment]
The error is fixed, you can now download 0.4.1 of Blazorise.Bootstrap package.
